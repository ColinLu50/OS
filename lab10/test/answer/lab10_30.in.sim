56577
1186
1184
982
1174
974
