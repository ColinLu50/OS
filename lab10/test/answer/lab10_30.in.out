FCFS
202 -> 438 -> 799 -> 172 -> 783 -> 799 -> 812 -> 102 -> 81 -> 242 -> 802 -> 982 -> 65 -> 64 -> 719 -> 643 -> 657 -> 593 -> 325 -> 266 -> 205 -> 321 -> 138 -> 425 -> 5 -> 43 -> 243 -> 927 -> 31 -> 883 -> 596 -> 551 -> 206 -> 680 -> 189 -> 858 -> 612 -> 583 -> 537 -> 453 -> 152 -> 233 -> 12 -> 237 -> 944 -> 429 -> 392 -> 903 -> 848 -> 708 -> 403 -> 831 -> 780 -> 471 -> 670 -> 647 -> 723 -> 728 -> 307 -> 349 -> 506 -> 208 -> 226 -> 895 -> 168 -> 103 -> 895 -> 979 -> 88 -> 532 -> 698 -> 637 -> 651 -> 831 -> 430 -> 857 -> 719 -> 670 -> 114 -> 402 -> 119 -> 881 -> 596 -> 881 -> 961 -> 514 -> 764 -> 110 -> 825 -> 123 -> 264 -> 860 -> 917 -> 325 -> 878 -> 648 -> 596 -> 84 -> 925 -> 686 -> 891 -> 572 -> 616 -> 368 -> 171 -> 292 -> 79 -> 34 -> 16 -> 527 -> 773 -> 550 -> 487 -> 254 -> 184 -> 192 -> 644 -> 674 -> 941 -> 782 -> 857 -> 365 -> 281 -> 275 -> 806 -> 131 -> 515 -> 757 -> 968 -> 127 -> 502 -> 660 -> 142 -> 418 -> 638 -> 112 -> 338 -> 807 -> 303 -> 716 -> 758 -> 171 -> 817 -> 971 -> 967 -> 441 -> 546 -> 817 -> 681 -> 370 -> 866 -> 714 -> 823 -> 802 -> 102 -> 701 -> 482 -> 974 -> 89 -> 313 -> 619 -> 609 -> 111 -> 505 -> 20 -> 574 -> 736 -> 626 -> 647 -> 70 -> 637 -> 844 -> 697 -> 903 -> 906 -> 326 -> 713 -> 713 -> 929 -> 461 -> 12 -> 772 -> 895 -> 481
56577
SSTF
202 -> 205 -> 206 -> 208 -> 192 -> 189 -> 184 -> 172 -> 171 -> 171 -> 168 -> 152 -> 142 -> 138 -> 131 -> 127 -> 123 -> 119 -> 114 -> 112 -> 111 -> 110 -> 103 -> 102 -> 102 -> 89 -> 88 -> 84 -> 81 -> 79 -> 70 -> 65 -> 64 -> 43 -> 34 -> 31 -> 20 -> 16 -> 12 -> 12 -> 5 -> 226 -> 233 -> 237 -> 242 -> 243 -> 254 -> 264 -> 266 -> 275 -> 281 -> 292 -> 303 -> 307 -> 313 -> 321 -> 325 -> 325 -> 326 -> 338 -> 349 -> 365 -> 368 -> 370 -> 392 -> 402 -> 403 -> 418 -> 425 -> 429 -> 430 -> 438 -> 441 -> 453 -> 461 -> 471 -> 481 -> 482 -> 487 -> 502 -> 505 -> 506 -> 514 -> 515 -> 527 -> 532 -> 537 -> 546 -> 550 -> 551 -> 572 -> 574 -> 583 -> 593 -> 596 -> 596 -> 596 -> 609 -> 612 -> 616 -> 619 -> 626 -> 637 -> 637 -> 638 -> 643 -> 644 -> 647 -> 647 -> 648 -> 651 -> 657 -> 660 -> 670 -> 670 -> 674 -> 680 -> 681 -> 686 -> 697 -> 698 -> 701 -> 708 -> 713 -> 713 -> 714 -> 716 -> 719 -> 719 -> 723 -> 728 -> 736 -> 757 -> 758 -> 764 -> 772 -> 773 -> 780 -> 782 -> 783 -> 799 -> 799 -> 802 -> 802 -> 806 -> 807 -> 812 -> 817 -> 817 -> 823 -> 825 -> 831 -> 831 -> 844 -> 848 -> 857 -> 857 -> 858 -> 860 -> 866 -> 878 -> 881 -> 881 -> 883 -> 891 -> 895 -> 895 -> 895 -> 903 -> 903 -> 906 -> 917 -> 925 -> 927 -> 929 -> 941 -> 944 -> 961 -> 967 -> 968 -> 971 -> 974 -> 979 -> 982
1186
SCAN
202 -> 192 -> 189 -> 184 -> 172 -> 171 -> 168 -> 152 -> 142 -> 138 -> 131 -> 127 -> 123 -> 119 -> 114 -> 112 -> 111 -> 110 -> 103 -> 102 -> 89 -> 88 -> 84 -> 81 -> 79 -> 70 -> 65 -> 64 -> 43 -> 34 -> 31 -> 20 -> 16 -> 12 -> 5 -> 0 -> 205 -> 206 -> 208 -> 226 -> 233 -> 237 -> 242 -> 243 -> 254 -> 264 -> 266 -> 275 -> 281 -> 292 -> 303 -> 307 -> 313 -> 321 -> 325 -> 326 -> 338 -> 349 -> 365 -> 368 -> 370 -> 392 -> 402 -> 403 -> 418 -> 425 -> 429 -> 430 -> 438 -> 441 -> 453 -> 461 -> 471 -> 481 -> 482 -> 487 -> 502 -> 505 -> 506 -> 514 -> 515 -> 527 -> 532 -> 537 -> 546 -> 550 -> 551 -> 572 -> 574 -> 583 -> 593 -> 596 -> 609 -> 612 -> 616 -> 619 -> 626 -> 637 -> 638 -> 643 -> 644 -> 647 -> 648 -> 651 -> 657 -> 660 -> 670 -> 674 -> 680 -> 681 -> 686 -> 697 -> 698 -> 701 -> 708 -> 713 -> 714 -> 716 -> 719 -> 723 -> 728 -> 736 -> 757 -> 758 -> 764 -> 772 -> 773 -> 780 -> 782 -> 783 -> 799 -> 802 -> 806 -> 807 -> 812 -> 817 -> 823 -> 825 -> 831 -> 844 -> 848 -> 857 -> 858 -> 860 -> 866 -> 878 -> 881 -> 883 -> 891 -> 895 -> 903 -> 906 -> 917 -> 925 -> 927 -> 929 -> 941 -> 944 -> 961 -> 967 -> 968 -> 971 -> 974 -> 979 -> 982
1184
C-SCAN
202 -> 192 -> 189 -> 184 -> 172 -> 171 -> 168 -> 152 -> 142 -> 138 -> 131 -> 127 -> 123 -> 119 -> 114 -> 112 -> 111 -> 110 -> 103 -> 102 -> 89 -> 88 -> 84 -> 81 -> 79 -> 70 -> 65 -> 64 -> 43 -> 34 -> 31 -> 20 -> 16 -> 12 -> 5 -> 0 -> 985 -> 982 -> 979 -> 974 -> 971 -> 968 -> 967 -> 961 -> 944 -> 941 -> 929 -> 927 -> 925 -> 917 -> 906 -> 903 -> 895 -> 891 -> 883 -> 881 -> 878 -> 866 -> 860 -> 858 -> 857 -> 848 -> 844 -> 831 -> 825 -> 823 -> 817 -> 812 -> 807 -> 806 -> 802 -> 799 -> 783 -> 782 -> 780 -> 773 -> 772 -> 764 -> 758 -> 757 -> 736 -> 728 -> 723 -> 719 -> 716 -> 714 -> 713 -> 708 -> 701 -> 698 -> 697 -> 686 -> 681 -> 680 -> 674 -> 670 -> 660 -> 657 -> 651 -> 648 -> 647 -> 644 -> 643 -> 638 -> 637 -> 626 -> 619 -> 616 -> 612 -> 609 -> 596 -> 593 -> 583 -> 574 -> 572 -> 551 -> 550 -> 546 -> 537 -> 532 -> 527 -> 515 -> 514 -> 506 -> 505 -> 502 -> 487 -> 482 -> 481 -> 471 -> 461 -> 453 -> 441 -> 438 -> 430 -> 429 -> 425 -> 418 -> 403 -> 402 -> 392 -> 370 -> 368 -> 365 -> 349 -> 338 -> 326 -> 325 -> 321 -> 313 -> 307 -> 303 -> 292 -> 281 -> 275 -> 266 -> 264 -> 254 -> 243 -> 242 -> 237 -> 233 -> 226 -> 208 -> 206 -> 205
982
LOOK
202 -> 192 -> 189 -> 184 -> 172 -> 171 -> 168 -> 152 -> 142 -> 138 -> 131 -> 127 -> 123 -> 119 -> 114 -> 112 -> 111 -> 110 -> 103 -> 102 -> 89 -> 88 -> 84 -> 81 -> 79 -> 70 -> 65 -> 64 -> 43 -> 34 -> 31 -> 20 -> 16 -> 12 -> 5 -> 205 -> 206 -> 208 -> 226 -> 233 -> 237 -> 242 -> 243 -> 254 -> 264 -> 266 -> 275 -> 281 -> 292 -> 303 -> 307 -> 313 -> 321 -> 325 -> 326 -> 338 -> 349 -> 365 -> 368 -> 370 -> 392 -> 402 -> 403 -> 418 -> 425 -> 429 -> 430 -> 438 -> 441 -> 453 -> 461 -> 471 -> 481 -> 482 -> 487 -> 502 -> 505 -> 506 -> 514 -> 515 -> 527 -> 532 -> 537 -> 546 -> 550 -> 551 -> 572 -> 574 -> 583 -> 593 -> 596 -> 609 -> 612 -> 616 -> 619 -> 626 -> 637 -> 638 -> 643 -> 644 -> 647 -> 648 -> 651 -> 657 -> 660 -> 670 -> 674 -> 680 -> 681 -> 686 -> 697 -> 698 -> 701 -> 708 -> 713 -> 714 -> 716 -> 719 -> 723 -> 728 -> 736 -> 757 -> 758 -> 764 -> 772 -> 773 -> 780 -> 782 -> 783 -> 799 -> 802 -> 806 -> 807 -> 812 -> 817 -> 823 -> 825 -> 831 -> 844 -> 848 -> 857 -> 858 -> 860 -> 866 -> 878 -> 881 -> 883 -> 891 -> 895 -> 903 -> 906 -> 917 -> 925 -> 927 -> 929 -> 941 -> 944 -> 961 -> 967 -> 968 -> 971 -> 974 -> 979 -> 982
1174
C-LOOK
202 -> 192 -> 189 -> 184 -> 172 -> 171 -> 168 -> 152 -> 142 -> 138 -> 131 -> 127 -> 123 -> 119 -> 114 -> 112 -> 111 -> 110 -> 103 -> 102 -> 89 -> 88 -> 84 -> 81 -> 79 -> 70 -> 65 -> 64 -> 43 -> 34 -> 31 -> 20 -> 16 -> 12 -> 5 -> 982 -> 979 -> 974 -> 971 -> 968 -> 967 -> 961 -> 944 -> 941 -> 929 -> 927 -> 925 -> 917 -> 906 -> 903 -> 895 -> 891 -> 883 -> 881 -> 878 -> 866 -> 860 -> 858 -> 857 -> 848 -> 844 -> 831 -> 825 -> 823 -> 817 -> 812 -> 807 -> 806 -> 802 -> 799 -> 783 -> 782 -> 780 -> 773 -> 772 -> 764 -> 758 -> 757 -> 736 -> 728 -> 723 -> 719 -> 716 -> 714 -> 713 -> 708 -> 701 -> 698 -> 697 -> 686 -> 681 -> 680 -> 674 -> 670 -> 660 -> 657 -> 651 -> 648 -> 647 -> 644 -> 643 -> 638 -> 637 -> 626 -> 619 -> 616 -> 612 -> 609 -> 596 -> 593 -> 583 -> 574 -> 572 -> 551 -> 550 -> 546 -> 537 -> 532 -> 527 -> 515 -> 514 -> 506 -> 505 -> 502 -> 487 -> 482 -> 481 -> 471 -> 461 -> 453 -> 441 -> 438 -> 430 -> 429 -> 425 -> 418 -> 403 -> 402 -> 392 -> 370 -> 368 -> 365 -> 349 -> 338 -> 326 -> 325 -> 321 -> 313 -> 307 -> 303 -> 292 -> 281 -> 275 -> 266 -> 264 -> 254 -> 243 -> 242 -> 237 -> 233 -> 226 -> 208 -> 206 -> 205
974
